src/python/WMCore/Services/PyCondor/PyCondorAPI.py

```diff
@@ -22,22 +22,31 @@ def __init__(self):
         self.schedd = htcondor.Schedd()
         self.coll = htcondor.Collector()
 
-    def getCondorJobs(self, constraint, attr_list):
+    def getCondorJobs(self, constraint='true', attrList=None, opts="SummaryOnly"):
```

Comment:
looking at this `opts="SummaryOnly"`, I realize you were already aware of the option path I was previously talking about. And it is quite good you added it here, so I would suggest to use it in order to implement my suggestion above. Thanks for that one -  I really like it!